### DISCUSSION TITLE
📚 Documentation: demo and commercializing API access

### DISCUSSION THREAD

**Discussion by DankMemeGuy** [appwrite/appwrite#4745] [General]
### 💭 Description

I have read the documentation, but I think there's 2 big issues that I cannot seem to find more about.

1. is there any demo of backend admin page? I know competitor software like PocketBase have https://pocketbase.io/demo/ so I think this would be invaluable to see if this is the right software for someone to use, before having to just implement it and find out if they like the UI, features in practice (one thing to read it, another to try it)
2. Is it possible to sell access to the API? So if I were to use AppWrite, then I wanted to let 3rd party developers use my site for their own backend, is there an ability for me to start charging for API usage? Based on a quota or membership tier? Or anything really? I can't see any info on the documentation about letting 3rd party developers use the API, I only see reference to the the main developer implementing the API.

Thank you!

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by DankMemeGuy** [comment]
Bump anyone?

**Comment by stnguyen90** [comment]
> 1. is there any demo of backend admin page?

Thanks for the suggestion! It sounds like a good idea, but I don't think we have the capacity to support a public demo instance at the moment. The good news is it's easy to test out Appwrite via our [1 command install](https://appwrite.io/docs/installation#installWithDocker) or by running Appwrite from [Gitpod](https://github.com/appwrite/integration-for-gitpod).

**Reply by DankMemeGuy** [reply]
> or by running Appwrite from [Gitpod](https://github.com/appwrite/integration-for-gitpod).

This took me less than a minute! Thank you. I think you should use this link as the link for a demo. Whether on the documentation page under a Demo heading, or on the website.

**Comment by stnguyen90** [comment]
> Is it possible to sell access to the API?

Appwrite was designed more as a backend for building apps rather than as an API platform. That said, you might be able to put Appwrite behind something like RapidAPI with an API Key.

**Reply by DankMemeGuy** [reply]
I'll try RapidAPI for that. I think it should really be looked to implement in the horizon though. As AppWrite matures, and the projects evolve, there will be a growing need/want for commercializing the API. Imagine someone creates a small web app utility that they could integrate with other websites if there was a way to commercialize the API, or an social type app where they could sell API access to other developers to allow other apps and services to be built off of/integrated with.

In my specific use case, I am doing a bulk data type project, think of it like a robust engine, and I was asked by other developers if I had public API that they could use to be able to build their own 'mini engines' that connect to my data pipeline.
